package.json

```diff
@@ -45,6 +45,8 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
```

Comment:
remove any styled-component entries in all package.json files since we no longer need them!